Gemfile.lock

```diff
@@ -1093,4 +1093,4 @@ RUBY VERSION
    ruby 3.0.5p211
 
 BUNDLED WITH
-   2.3.22
```

Comment:
yes

Reply:
For future reference, this should've been removed; because this is an autogenerated file, that extra newline will be automatically added back to the version of this file that exists on all our servers, and we'll then get a series of git merge conflicts when we try to deploy: https://codedotorg.slack.com/archives/C03CM903Y/p1689289690572599